### ISSUE TITLE
Assets SVN repo domain has lapsed

### ISSUE THREAD

**Issue by pscollins** [Revolutionary-Games/Thrive#424]
Following the build instructions:

```
➜  Thrive git:(master) svn co http://crovea.net/svn/thrive_assets/ ./assets
svn: E175009: Unable to connect to a repository at URL 'http://crovea.net/svn/thrive_assets'
svn: E175009: XML Parsing failed: Unexpected root element 'html'
```

**Comment by Moopli** [comment]
We know, someone was supposed to update the docs -- our new repo is at [assets.revolutionarygamesstudio.com](assets.revolutionarygamesstudio.com), or at least it should be, as I haven't had a chance to check. Sorry!

**Comment by Oliveriver** [comment]
Don't know why that links to a GitHub page. The repo should be here: assets.revolutionarygamesstudio.com

**Comment by jjonj** [comment]
I had forgotten to push my update, now fixed with c15f7913c0ef
